internal/wrappers/pr.go

```diff
@@ -34,8 +34,24 @@ type AzurePRModel struct {
 	APIURL    string     `json:"apiUrl"`
 }
 
+type BitbucketCloudPRModel struct {
+	ScanID    string     `json:"scanId"`
+	ScmToken  string     `json:"scmToken"`
+	Namespace string     `json:"namespace"`
+	RepoName  string     `json:"repoName"`
+	PRID      int        `json:"prId"`
+	Policies  []PrPolicy `json:"violatedPolicyList"`
+}
+
+type BitbucketServerPRModel struct {
+	ScanID     string     `json:"scanId"`
+	ScmToken   string     `json:"scmToken"`
+	ServerURL  string     `json:"apiUrl"`
+	ProjectKey string     `json:"namespace"`
+	RepoName   string     `json:"repoName"`
+	PRID       int        `json:"prNumber"`
+	Policies   []PrPolicy `json:"violatedPolicyList"`
+}
 type PRWrapper interface {
-	PostPRDecoration(model *PRModel) (string, *WebError, error)
-	PostGitlabPRDecoration(model *GitlabPRModel) (string, *WebError, error)
-	PostAzurePRDecoration(model *AzurePRModel) (string, *WebError, error)
+	PostPRDecoration(model interface{}) (string, *WebError, error)
```

Comment:
Check please if the param can be Interface the representing PRModel, and all the PR models structs will extends it